core/src/main/java/ai/timefold/solver/core/impl/score/director/InnerScoreDirector.java

```diff
@@ -341,6 +341,11 @@ default Solution_ cloneWorkingSolution() {
     @Override
     void close();
 
+    /**
+     * This function clears all listener events that have been generated without triggering any of them.
+     */
+    void clearVariableListenerEvents();
```

Comment:
Maybe I'll be good if this method exists only the Bavet score director?
We don't need it anywhere else, because constraint verifier doesn't support anything else.

It should definitely have a Javadoc that clearly states what its purpose is and how destructive it is.